easybuild/framework/easyconfig/default.py

```diff
@@ -158,6 +158,11 @@
 
     # MODULES easyconfig parameters
     'whatis': [None, "List of brief (one line) package description entries", MODULES],
+    'usage': [None, "Usage instructions for the software", MODULES],
+    'docpaths': [None, "List of paths of installed documentation relative to package root dir", MODULES],
+    'docurls': [None, "List of URLs to package documentation", MODULES],
+    'support': [None, "Package-specific support/bug report address", MODULES],
+    'contact': [None, "Site contact for the package", MODULES],
```

Comment:
What about `upstream_contact` and `site_contact`?  This should make it clear.  And since the list values are free-form text fields, one could do something like
```
upstream_contact = [
    'Support e-mail: support@project.org',
    'Mailing list: discuss@project.org',
    'Bugtracker: https://www.project.org/bugzilla'
]
site_contact = ['Jane Admin']
```
Which would be rendered as
```
More information
================
 - Homepage: https://www.project.org
 - Upstream contact:
    - Support e-mail: support@project.org
    - Mailing list: discuss@project.org
    - Bugtracker: https://www.project.org/bugzilla
 - Site contact: Jane Admin
```